dev/tests/integration/testsuite/Magento/Cron/Model/ScheduleTest.php

```diff
@@ -60,14 +70,15 @@ public function testTryLockJobAlreadyLockedFails()
     public function testTryLockJobAlreadyLockedSucceeds()
     {
         $offsetInThePast = 2*24*60*60;
+        $gmtTimestamp = $this->dateTime->gmtTimestamp();
 
         $oldSchedule = $this->scheduleFactory->create()
             ->setCronExpr("* * * * *")
             ->setJobCode("test_job")
             ->setStatus(Schedule::STATUS_RUNNING)
-            ->setCreatedAt(strftime('%Y-%m-%d %H:%M:%S', $this->dateTime->gmtTimestamp() - $offsetInThePast))
-            ->setScheduledAt(strftime('%Y-%m-%d %H:%M', $this->dateTime->gmtTimestamp() - $offsetInThePast + 60))
-            ->setExecutedAt(strftime('%Y-%m-%d %H:%M', $this->dateTime->gmtTimestamp() - $offsetInThePast + 61));
+            ->setCreatedAt($this->getTimeFormat($gmtTimestamp - $offsetInThePast))
```

Comment:
Is it possible to use \Magento\Framework\Stdlib\DateTime\DateTime::gmtDate() ?
something like 
`$this->dateTime->gmtDate('Y-m-d H:i', $this->dateTime->gmtTimestamp() - $offsetInThePast);`
or
`        date('Y-m-d H:i', $this->dateTime->gmtTimestamp() - $offsetInThePast);`
should be enough for tests

Reply:
@xmav, code is fixed, please check if everything is well.

Thank you